charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/csi-azuredisk-scheduler-extender.yaml

```diff
@@ -32,7 +32,8 @@ kind: Deployment
 metadata:
   name: csi-azuredisk-scheduler-extender
   namespace: {{ .Release.Namespace }}
-  labels:
+  labels: 
+{{ include "azuredisk.labels" . | indent 4 }}
     app: csi-azuredisk-scheduler-extender
```

Comment:
We should probably allow for customization of the scheduler extender app name like the chart does for the controller app name either in this PR or a separate one.